Assets/Prefabs/Moveable/HeavyBlock.prefab

```diff
@@ -108,7 +108,7 @@ Rigidbody:
   m_PrefabAsset: {fileID: 0}
   m_GameObject: {fileID: 100000}
   serializedVersion: 2
-  m_Mass: 2
+  m_Mass: 420
```

Comment:
The actual change was requested by Kozzy and Lucy as the found the mass of these objects too light. It might seem a lot but it needs to be taken in relative consideration - the agent has a large mass by default. I've tested the objects with physics-based runs and it feels more appropriate.